ghcid-ng/src/ghci/parse/module_set.rs

```diff
@@ -0,0 +1,81 @@
+use std::collections::HashSet;
+
+use camino::Utf8Path;
+use camino::Utf8PathBuf;
+use miette::Context;
+use miette::IntoDiagnostic;
+
+use super::Module;
+
+/// A collection of source paths, retaining information about loaded modules in a `ghci`
+/// session.
+#[derive(Debug, Clone, Default, PartialEq, Eq)]
+pub struct ModuleSet {
+    map: HashSet<Utf8PathBuf>,
```

Comment:
maybe call this field `set`?

Reply:
Oh, good idea (a long time ago, this value was a map).